mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -181,7 +188,7 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
         xlabel = "Time [years]"
         ylabel = "[x$10^{22}$ J]"
 
-        if ref_casename_v0 != "None":
+        if ref_casename_v0 != 'None':
             figname = "{}/ohc_{}_{}_{}.png".format(plots_dir,
```

Comment:
we have many places in (all) the scripts where we are still using double quotes.
Is there a python best practice as far as quotes go?